### PR TITLE
2023-05-11-icd10cm_cause_claim_mapper_en

### PR DESCRIPTION
**Pull request by jsl-models**
This pretrained model maps ICD10CM codes, subsequently providing corresponding causes and generating claim analysis codes for each respective ICD10CM code.

### CODE REVIEW

File: docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+supported: true
+annotator: ChunkMapperModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained model maps ICD10CM codes, subsequently providing corresponding causes and generating claim analysis codes for each respective ICD10CM code.
```
Comment (muhammetsnts) [inline docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md:21]: ICD10CM should be ICD-10-CM

File: docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+|D69.51|Adverse effects of medical treatment|icd10cm_cause              |
+|D69.51|D69.51                              |icd10cm_claim_analysis_code|
+|D69.51|D69.51                              |icd10cm_claim_analysis_code|
+|G43.83|Headache disorders                  |icd10cm_cause              |
+|G43.83|Migraine                            |icd10cm_cause              |
+|G43.83|Tension-type headache               |icd10cm_cause              |
+|G43.83|G43.83                              |icd10cm_claim_analysis_code|
+|G43.83|G43.83                              |icd10cm_claim_analysis_code|
+|G43.83|G43.83                              |icd10cm_claim_analysis_code|
+|Z83.51|NONE                                |null                       |
```
Comment (muhammetsnts) [inline docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md:102]: change here. And also try to find some examples that icd10cm codes are not the same icd10cm claim analysis code.

File: docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+val document_assembler = new DocumentAssembler()
+       .setInputCol("text")
+       .setOutputCol("document")
+
+val chunk_assembler = new Doc2Chunk()
+      .setInputCols("document")
+      .setOutputCol("icd_chunk")
+
+val chunkerMapper = ChunkMapperModel
+       .pretrained("icd10cm_cause_claim_mapper", "en", "clinical/models")
```
Comment (muhammetsnts) [inline docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md:73]: please move this line to the end of ChunkMapperModel

File: docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md (hunk 1 above)
Comment (muhammetsnts) [reply docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md:21]: Add as a new sentence what if there is no claim code of an icd10cm code. (It returns None)

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md (hunk 1 above)
Comment (SKocer) [reply docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md]: Done.

File: docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md (hunk 3 above)
Comment (SKocer) [reply docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md]: Done.

**Review by muhammetsnts — APPROVED** [review]

**Comment by muhammetsnts** [comment]
Great job 💯

**Review by muhammetsnts — APPROVED** [review]

**Pull request merged**